src/Model/User.php

```diff
@@ -96,6 +96,59 @@ class User
 
 	private static $owner;
 
+	/**
+	 * Sends a welcome e-mail to the specified user
+	 *
+	 * @param array $user The user
+	 * @param string $password The user's password
+	 * @param string $preamble The user's password
+	 *
+	 * @return NULL|boolean from notification() and email() inherited
+	 * @throws HTTPException\InternalServerErrorException
+	 */
+	private static function send_welcome_email($user, $password, $preamble)
```

Comment:
```suggestion
	private static function sendWelcomeEmail(array $user, string $password, string $preamble)
```

 @art4 I'd like to introduce `HiddenString` in an upcoming PR, see #7244, for all these `password` strings, flying all around in this class ;-)

Reply:
Right! Done. :-)

Reply:
@nupplaphil interesting, I didn't realize we have the HiddenString lib installed. At least we can mark the password with the [SensitiveParameter](https://www.php.net/manual/en/class.sensitiveparameter.php) attribute right now.